source/index.blade.php

```diff
@@ -108,47 +108,7 @@ class="shape shape-2"
     <!-- ====== About End ====== -->
 
      <!-- ====== Testimonials Start ====== -->
```

Comment:
Very nice to have a partial to testimonial cart. Congratulations by this idea.

```suggestion
```

I think that we can remove this comment now. This could be made in a new PR to don't block this PR.